config/routes.rb

```diff
@@ -32,6 +32,8 @@
   get '/openid_connect/authorize' => 'openid_connect/authorization#index'
   get '/openid_connect/logout' => 'openid_connect/logout#index'
 
+  get '/no_js/css' => 'no_js#css'
```

Comment:
It's throwing me off a little to have an action named `#css`

WDYT of....
```suggestion
  get '/no_js/test(.:format)' => 'no_js#index', constraints: lambda { |req| req.format == :css }
```
(or some other way to make the `.css` an extension?)

Reply:
No I just forgot that we could hardcode it instead of rails adding `(.:format)` implicitly 🤦 

But yes same result

Reply:
On the specific path name, "test" may not be an accurate description, since the test has already happened by the time the resource is loaded. The intent is more to "mark" or "tag" the session. Thinking one of those names, if not too invasive-sounding (sorta avoiding the "track" or "fingerprint" kind of verbiage).

Reply:
I ended up just going with "track". I was gonna use "tag", but it gets kinda confusing talking about "tag" in the context of the "stylesheet tag" 😵